src/analytics/InjectTraits.ts

```diff
@@ -0,0 +1,28 @@
+import { PlatformPlugin, PluginType, SegmentEvent } from '@segment/analytics-react-native'
+
+/**
+ * Plugin that injects the user traits to every event
+ *
+ * Added to migrate to Segment v2 without breaking existing dashboards that rely on context_traits_*. All this data is
+ *  available as "super" properties, so we can eventually get rid of it.
+ *
+ * TODO eventually: remove dependencies and delete this plugin, see https://linear.app/valora/issue/ENG-60/remove-dependencies-on-context-traits-
+ */
+export class InjectTraits extends PlatformPlugin {
```

Comment:
How was this data previously being logged to segment? Was this happening automatically under the hood/is the removal of this behavior from segment v1 to v2 documented anywhere?

Reply:
It was done automatically. Here is the documentation for it https://github.com/segmentio/analytics-react-native/blob/c3e41ead9de261bd3232305444eccd824294acc7/MIGRATION_GUIDE.md#context-traits